src/main/java/seedu/address/model/person/Address.java

```diff
@@ -37,6 +37,14 @@ public static boolean isValidAddress(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Compares an address with another address.
+     * @param address The other address being compared.
+     */
+    public int compareTo(Address address) {
+        return this.value.compareTo(address.value);
+    }
+
```

Comment:
I think according to the code standards, we can omit JavaDoc for overriding methods and getters and setters. I feel like this is too time consuming.

Reply:
Indeed. However, I did not make the class implementing the comparator interface, hence the compareTo() method here is not overriding anything.
I would consider to make the class implements the interface if time permits.